File: src/main/java/Homework7/Drink.py
class Drink():
    def __init__(self, name, valume):
        self.__name = name
        self.__valume = valume

    def getName(self):
        return self.__name

    def setName(self, name):
        self.__name = name

    def getValume(self):
        return self.__valume

    def setValume(self, valume):
        self.__valume = valume

    def __str__(self):
        return f"Название: {self.__name}, стоимость: {self.__valume}"

File: src/main/java/Homework7/Drink.java
package Homework4;

public class Drink {
    private String name;
    private int value;

    public Drink(String name, int value) {
        this.name = name;
        this.value = value;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public float getVolume() {
        return value;
    }

    public void setVolume(int value) {
        this.value = value;
    }

    public String toString() {
        return String.format("Название: %s; Стоимость: %d", name, value);
    }
}
